src/app/pages/OnDemandTvPage/index.jsx

```diff
@@ -25,19 +28,63 @@ const getGroups = (zero, one, two, three, four, five) => ({
   group5: five,
 });
 
+const getEpisodeAvailability = (availableFrom, availableUntil) => {
```

Comment:
This seems like it's duplicated from the On Demand Radio Page - is there a way in which we could refactor & reuse?

Reply:
I did consider this. However, I believe DRY should be less about duplicate code existing and more about what it is affecting. So having two functions that check the availability of the same radio episode would break the DRY principle as you would need to remember to update both functions if you changed the behaviour. For this function in the PR, it does not represent the same 'knowledge area' as the OD Radio function does. If we change the behaviour of one function, we don't expect to change the behaviour of the other.